backend/sample.env

```diff
@@ -86,9 +86,9 @@ PROMPT_STUDIO_FILE_PATH=/app/prompt-studio-data
 
 # Structure Tool Image (Runs prompt studio exported tools)
 # https://hub.docker.com/r/unstract/tool-structure
-STRUCTURE_TOOL_IMAGE_URL="docker:unstract/tool-structure:0.0.42"
+STRUCTURE_TOOL_IMAGE_URL="docker:unstract/tool-structure:0.0.43"
```

Comment:
@Deepak-Kesavan was this version built? I don't see [this built recently](https://github.com/Zipstack/unstract/actions/workflows/docker-tools-build-push.yaml)

Reply:
I usually build from main only.